spec.emu

```diff
@@ -932,7 +932,7 @@ location: https://github.com/tc39/proposal-decimal/
             1. <ins>If _value_ is an object that has an [[Decimal128Data]] internal slot, then</ins>
               1. <ins>Let _d_ be _value_.[[Decimal128Data]].</ins>
               1. <ins>Let _serialized_ be Decimal128ToExponentialString(_d_, *false*).</ins>
-              1. <ins>Return ! ToNumber(_serialized_).</ins>
+              1. <ins>Let _n be ! ToNumber(_serialized_).</ins>
             1. Let _prim_ be ? ToNumeric(_value_).
             1. If _prim_ is a BigInt, let _n_ be 𝔽(ℝ(_prim_)).
             1. Otherwise, let _n_ be _prim_.
```

Comment:
These three steps need to be in an `Else,` block.

Reply:
done